packages/api-client/src/api/getProduct/defaultQuery.ts

```diff
@@ -0,0 +1,38 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  query product ($id: ID, $slug: String) {
+    product(id: $id, slug: $slug) {
+      slug
+      description
+      variants {
+        id
+        sku
+        name
+        price
+        priceWithTax
+        currencyCode
+      }
+      featuredAsset {
+        source
```

Comment:
use the `preview` property any time you get an Asset - the `source` is the source binary and might be very large, whereas `preview` is constrained to a sensible file size.